### DISCUSSION TITLE
Add spacing between a title and the text below

### DISCUSSION THREAD

**Discussion by annahtosahs** [typst/typst#3396] [General]
### Description

![Screenshot from 2024-02-11 11-32-21](https://github.com/typst/typst/assets/118920696/70094b61-6bd5-4263-9153-a6b961100160)
![Screenshot from 2024-02-11 11-30-46](https://github.com/typst/typst/assets/118920696/d18a1847-f478-42db-afb8-339733cfbabe)

i think the difference is as clear as day.
also shouldn't computer modern be the default font, the first one felt weird.

I suggest the overall styling of typst be similar if not identical to Latex, as the developers stated on their website, they didn't leave Latex because of it's styling but rather because of its UX and i agree, as of now, Latex documents look better than typst, and i see no reason to re-invent the wheel with styling.

### Use Case

This can help latex feel at home, everything with the tool is very good and promising but the styling feels inferior to Latex

**Comment by Enivex** [comment]
(New) computer modern math is already the default math font. It's not clear to me whether you're talking about the text font, but that's largely a matter of personal preference.

Regarding spacing, that can be adjusted with a show rule.

I'd recommend you check out the "quick-questions' channel on the Discord. GitHub issues aren't the appropriate venue.

**Comment by xkevio** [comment]
The "inferior styling" is just that -- a default you can change. As was said, the math font is the same, the text font is "Linux Libertine" which can be changed with one line and is personal preference. As for the spacing, that is also a very simple change depending on if `Definition` is a heading element or not. And, fwiw, this guide will help you achieve the LaTeX look again: https://typst.app/docs/guides/guide-for-latex-users/#latex-look
